docs/transit.md

```diff
@@ -211,11 +209,9 @@ from wormhole.twisted.transit import TransitSender
 @inlineCallbacks
 def do_transit():
     s = TransitSender(relay)
-    my_relay_hints = s.get_relay_hints()
-    my_direct_hints = yield s.get_direct_hints()
+    my_connection_hints = s.get_connection_hints()
```

Comment:
it looks like `get_connection_hints()` returns a Deferred (since it's `@inlineCallbacks`), let's add a `yield` here